app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,40 @@
+require 'dotenv'
+Dotenv.load
```

Comment:
this should be loaded in an initializer, or a config, not directly in the controller.